drake/examples/Cars/simulateLCM.cpp

```diff
@@ -78,11 +89,22 @@ int main(int argc, char* argv[]) {
   // be reused
   DrakeJoint::FloatingBaseType floating_base_type = DrakeJoint::QUATERNION;
 
-
   auto rigid_body_sys = make_shared<RigidBodySystem>();
-  rigid_body_sys->addRobotFromFile(argv[1], floating_base_type);
-  auto const & tree = rigid_body_sys->getRigidBodyTree();
-  for (int i=2; i<argc; i++)
+
+  // The following variable, weld_to_frame, is only needed if the model is a
+  // URDF file. It is needed since URDF does not specify the location and
+  // orientation of the car's root node in the world. If the model is an SDF,
+  // weld_to_frame is ignored by the parser.
```

Comment:
I agree to postpone the discussion about whether `RigidBodyTree` should represent a single robot in the system or all of the robots in the system in a separate issue / PR.

Regarding the question of whether a generalized `addRobotFromFile()` should be supported, I am now in favor of it but would like to modify both the API and semantics slightly. Here is my new proposal:

```
void AddRobotFromFile(
      const std::string& file_name,
      const std::string& robot_name,
      const DrakeJoint::FloatingBaseType floating_base_type =
          DrakeJoint::QUATERNION,
      const Eigen::Vector3d floating_base_xyz = Eigen::Vector3d::Zero(),
      const Eigen::Vector3d floating_base_rpy = Eigen::Vector3d::Zero());
```

The `robot_name` parameter is necessary since an SDF file may contain multiple robot, each robot may need to be placed in the world at a different position and orientation, and the user may not want to add every robot in the SDF into the system.

The `floating_base_rpy` and `floating_base_xyz` parameters specify how the robot is positioned in the world. If the file is an SDF and the pose of the model in the world is already specified, throw an exception. In other words, only allow the pose of the robot to be specified once, either through the `floating_base_rpy` and `floating_base_xyz` parameters or through the `<pose>` element in the SDF model, but not both.

Thoughts?